doc/source/turkey.md

```diff
@@ -0,0 +1,33 @@
+# 🇹🇷 Turkey
+
+Turkey, officially known as the Republic of Turkey, is the country which is the bridge between Europe and Asia. It shares borders with 8 different countries and 3 seas. It consists of 7 geographical regions and 81 cities. The region on which the country is situated is called Anatolia and accepted as a peninsula. Anatolia has been a strategically important area throughout the history. Turkish people came to Anatolia in 1071 first and since then they are living there. Never mind, let’s focus on the country again.
```

Comment:
Turkey, officially known as the Republic of Turkey, is the country that is the bridge between Europe and Asia. It shares borders with 8 different countries and 3 seas. It consists of 7 geographical regions and 81 cities. The region in which the country is situated is called Anatolia and is accepted as a peninsula. Anatolia has been a strategically important area throughout history. Turkish people came to Anatolia in 1071 first and since then they are living there. Never mind, let’s focus on the country again.